<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>74</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="verticalSpacing">
    <number>5</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item row="2" column="0" colspan="3">
    <widget class="QComboBox" name="combo_mode"/>
   </item>
   <item row="3" column="1">
    <widget class="QToolButton" name="button_flip">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">padding: 0px</string>
     </property>
     <property name="text">
      <string>â‡„</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLineEdit" name="value_lower"/>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QComboBox" name="combo_attribute"/>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="value_upper"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
